app/core/config/application.js

```diff
@@ -1,35 +1,11 @@
 // @flow
 
-import {
-  API_KEY_GOOGLE_MAPS,
-  AUTH_ANDROID_GOOGLE_CLIENTID,
-  AUTH_IOS_GOOGLE_CLIENTID,
-  AUTH_KIWI_BACKEND,
-  SENTRY_DSN,
-} from 'react-native-dotenv';
+import { SENTRY_DSN } from 'react-native-dotenv';
 
 const application = {
   animations: {
     duration: 100, // ms
   },
-  apiKey: {
-    googleMaps: sanitize(API_KEY_GOOGLE_MAPS),
-  },
-  auth: {
-    android: {
-      google: {
-        clientId: sanitize(AUTH_ANDROID_GOOGLE_CLIENTID),
-      },
-    },
-    ios: {
-      google: {
-        clientId: sanitize(AUTH_IOS_GOOGLE_CLIENTID),
-      },
-    },
-    kiwi: {
-      backend: sanitize(AUTH_KIWI_BACKEND),
```

Comment:
What were these for?

Reply:
I don't think they ever were used

Reply:
At least for as long as I have been in the project